CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 2.33.0-rc.0 / 2022-01-12
+
+* [CHANGE] PromQL: Promote negative offset and `@` modifer to stable features. #10121
+* [CHANGE] Web: Promote remote-write-receiver to stable. #10119
+* [FEATURE] Config: Add `stripPort` template function. #10002
+* [FEATURE] Promtool: Add cardinality analysis to `check metrics`, enabled by flag `--extended`. #10045
+* [FEATURE] SD: Enable target discovery in own K8s namespace. #9881
+* [FEATURE] SD: Add provider ID label in K8s SD. #9603
+* [FEATURE] Web: Add limit to the rules api. #10152
```

Comment:
It should be specified that `limit` is a field, not a general limit for the response. I saw this exact thing coming last night and I don't know why I didn't change the commit title, sorry about that.

```suggestion
* [FEATURE] Web: Add limit field to the rules API. #10152
```